lib/rules/disallow-capitalized-comments.js

```diff
@@ -57,7 +57,7 @@ module.exports.prototype = {
         var letterPattern = require('../../patterns/L');
         var lowerCasePattern = require('../../patterns/Ll');
 
-        file.getComments().forEach(function(comment) {
+        file.iterateTokensByType(['Line', 'Block'], function(comment) {
```

Comment:
How do we actually do this? I don't see where the types for those tokens are being defined.